src/Common/src/Interop/Windows/BCrypt/Interop.BCryptImportKey.cs

```diff
@@ -0,0 +1,57 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+
+using Microsoft.Win32.SafeHandles;
```

Comment:
I think SafeAlgorithmHandle is in the Internal.NativeCrypto namespace.

```suggestion
using Internal.NativeCrypto;
using Microsoft.Win32.SafeHandles;
```